userguide/sharing.rst

```diff
@@ -1004,36 +863,25 @@ some troubleshooting tips. Reading through this entire chapter before
 creating any SMB shares is recommended to gain a better understanding of
 the configuration scenario that meets the specific network requirements.
 
-.. warning:: SMB1 is disabled by default for security reasons. If legacy
-   clients are unable to connect to the share, open :ref:`Shell`, type
-   :command:`sysctl freenas.services.smb.config.server_min_protocol=NT1`,
-   then restart the :ref:`SMB` service. If that resolves the issue, go to
-   :ref:`Tunables` and creating a tunable with a  :guilabel:`Variable` of
-   *freenas.services.smb.config.server_min_protocol*, a
-   :guilabel:`Value` of *NT1*, and a :guilabel:`Type` of *Sysctl*.
+`SMB Tips and Tricks <https://forums.freenas.org/index.php?resources/smb-tips-and-tricks.15/>`__
+shows helpful hints for configuring and managing SMB networking.
+The
+`FreeNAS and Samba (CIFS) permissions <https://www.youtube.com/watch?v=RxggaE935PM>`__
+and
+`Advanced Samba (CIFS) permissions on FreeNAS <https://www.youtube.com/watch?v=QhwOyLtArw0>`__
+videos clarify setting up permissions on SMB shares. Another helpful
+reference is
+`Methods For Fine-Tuning Samba Permissions <https://forums.freenas.org/index.php?threads/methods-for-fine-tuning-samba-permissions.50739/>`__.
 
-.. tip:: `SMB Tips and Tricks
-   <https://forums.freenas.org/index.php?resources/smb-tips-and-tricks.15/>`__
-   shows helpful hints for configuring and managing SMB networking.
-   The `FreeNAS and Samba (CIFS) permissions
-   <https://www.youtube.com/watch?v=RxggaE935PM>`__
-   and
-   `Advanced Samba (CIFS) permissions on FreeNAS
-   <https://www.youtube.com/watch?v=QhwOyLtArw0>`__
-   videos clarify setting up permissions on SMB shares. Another
-   helpful reference is
-   `Methods For Fine-Tuning Samba Permissions
-   <https://forums.freenas.org/index.php?threads/methods-for-fine-tuning-samba-permissions.50739/>`__.
-
-
-.. tip:: Run :command:`smbstatus` from the :ref:`Shell` for a list of
-   active connections and users.
+.. warning:: SMB1 is disabled by default for security. If legacy
+   clients are unable to connect to the share, go to
```

Comment:
Replace all of this with "If necessary, SMB1 can be enabled in Services --> SMB Settings."
Remove the rest.